lib/mp4/transmuxer.js

```diff
@@ -891,17 +892,10 @@ CoalesceStream.prototype.flush = function(flushSource) {
 
   this.emittedTracks += this.pendingTracks.length;
 
-  initSegment = mp4.initSegment(this.pendingTracks);
+  event.serializedTracks = mp4Utils.serializeTracks(this.pendingTracks);
```

Comment:
This may not be the best method for passing the data out of the worker, but the data seemed small enough that it wouldn't make a big difference in performance, and was relatively straightforward. I'd definitely be open to alternative approaches though.